citadel/indico_citadel/search.py

```diff
@@ -41,7 +41,7 @@ def active(cls):
         return bool(CitadelPlugin.settings.get('search_backend_url') and
                     CitadelPlugin.settings.get('search_backend_token'))
 
-    def search(self, query, user=None, page=1, object_types=(), **params):
+    def search(self, query, user=None, page=1, object_types=(), allow_admin=False, **params):
```

Comment:
I'd still prefer `admin_override_enabled` here. `allow_admin` is usually used to indicate whether to take *potential* admin access into account. but if this is set to true here, it already means that there's a user using their admin access.